pkg/controller/efs/filesystem/setup.go

```diff
@@ -96,12 +146,41 @@ func postObserve(_ context.Context, cr *svcapitypes.FileSystem, obj *svcsdk.Desc
 	return obs, nil
 }
 
-func preUpdate(_ context.Context, cr *svcapitypes.FileSystem, obj *svcsdk.UpdateFileSystemInput) error {
+func (e *custom) preUpdate(ctx context.Context, cr *svcapitypes.FileSystem, obj *svcsdk.UpdateFileSystemInput) error {
 	obj.FileSystemId = pointer.ToOrNilIfZeroValue(meta.GetExternalName(cr))
 	// Type of this field is *float64 but in practice, only integer values are allowed.
 	if cr.Spec.ForProvider.ProvisionedThroughputInMibps != nil {
 		obj.ProvisionedThroughputInMibps = aws.Float64(float64(pointer.Int64Value(cr.Spec.ForProvider.ProvisionedThroughputInMibps)))
 	}
+
+	if e.cache.backupPolicy != *cr.Spec.ForProvider.Backup {
+		var policy *string
+		if pointer.BoolValue(cr.Spec.ForProvider.Backup) {
+			policy = aws.String("ENABLED")
+		} else {
+			policy = aws.String("DISABLED")
+		}
+
+		if pointer.StringValue(obj.FileSystemId) == "" {
+			obj.FileSystemId = pointer.ToOrNilIfZeroValue(meta.GetExternalName(cr))
+		}
+		_, err := e.client.PutBackupPolicyWithContext(ctx,
+			&svcsdk.PutBackupPolicyInput{
+				FileSystemId: obj.FileSystemId,
+				BackupPolicy: &svcsdk.BackupPolicy{Status: policy}})
+		if err != nil {
+			return err
+		}
+	}
+
+	/*	Backup is not a direct part of the FileSystem, but an externally managed BackupPolicy.
+		Changing only the backup policy will result in an empty request and thus an Error.
+		To avoid this, we return an error if the BackupPolicy is the only thing that has changed.
+	*/
+	if ptr.Equal(cr.Spec.ForProvider.ThroughputMode, obj.ThroughputMode) && pointer.Int64Value(cr.Spec.ForProvider.ProvisionedThroughputInMibps) == int64(aws.Float64Value(obj.ProvisionedThroughputInMibps)) {
+		return errors.New("throughputmode and provisionedthroughputinmibps are unchanged, aborting update")
```

Comment:
Returning an error here will trigger a constant reschedule of the reconcile and make the resource show as unready - although there is actual not any error. If we really would want to prevent the update we would need to modify the `IsResourceUpToDate` result in `Observe`.